### PR TITLE
[zh] Fix and improve application failover

### PR DESCRIPTION
**Pull request by wilsonwu**
**What type of PR is this?**

/kind documentation

**What this PR does / why we need it**:
Fix format and typo in Chinese doc.

**Which issue(s) this PR fixes**:
No

**Special notes for your reviewer**:

### CODE REVIEW

**Review by samzong — COMMENTED** [review]
Thanks for your contribution! 👍

It seems that parts of commits in your PR that don't belong to you. This may have been caused by an improper rebase, and you might need to fix it, pls.

**Review by Arhell — COMMENTED** [review]
Please  signed off
https://github.com/karmada-io/website/pull/442/checks?check_run_id=15656594291

File: i18n/zh/docusaurus-plugin-content-docs/current/userguide/failover/application-failover.md [hunk 1]
```diff
@@ -191,7 +194,6 @@ spec:
 
 :::note
 
-应用故障转移的开发仍在进行中。我们正在收集用户案例。如果您对此功能感兴趣，请随时开启一个 enhancement issue 让我们知道。
+应用故障转移的开发仍在进行中。我们正在收集用户案例。如果您对此功能感兴趣，请随时开启一个 Enhancement Issue 让我们知道。
```
Comment (samzong) [inline i18n/zh/docusaurus-plugin-content-docs/current/userguide/failover/application-failover.md:197]: 建议统一下 "应用级故障转移" or "应用故障转移"

Comment (wilsonwu) [reply i18n/zh/docusaurus-plugin-content-docs/current/userguide/failover/application-failover.md]: Fixed all application fallover to "应用故障迁移"

**Comment by Poor12** [comment]
LGTM for the content

**Comment by XiShanYongYe-Chang** [comment]
Hi @wilsonwu, can you help squash commits into one?

**Review by XiShanYongYe-Chang — COMMENTED** [review]
It seems that some files have been mistakenly deleted.

**Comment by wilsonwu** [comment]
> It seems that some files have been mistakenly deleted.

Some problems for this branch, I will resubmit a PR for this change.

**Pull request closed without merging**